### PR TITLE
App download v2

### LINKED ISSUE #40: App Advertise Above Footer
image

### PR DESCRIPTION
**Pull request by ravverma**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #40 

Test URLs:
- Before: https://main--icicidirect--aemsites.hlx.live/research/equity
- After: https://app-download-v2--icicidirect--aemsites.hlx.live/research/equity

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------
 app-download-v2--icicidirect--aemsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/app-download&index=0)
image

### CODE REVIEW

**Review by sdmcraft — COMMENTED** [review]
Can we have the images directly placed in the word doc?

**Comment by ravverma** [comment]
> Can we have the images directly placed in the word doc?

those are small badge files not getting added to the doc, I have downloaded officially from google badge https://play.google.com/intl/en_us/badges/

File: blocks/app-download/app-download.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+.block.app-download h3 {
+  font-weight: var(--font-weight-medium);
+  padding-bottom: 1.25rem;
+  text-align: center;
+  font-size: 1.125rem;
+}
+
+.block.app-download > div p {
+  font: var(--font-weight-regular) 1rem / 1.5rem Poppins, sans-serif;
```
Comment (anagarwa) [inline blocks/app-download/app-download.css:28]: @ravverma fallback is "Poppins Fallback"

File: blocks/app-download/app-download.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  flex: 0 0 8.75rem;
+  max-width: 8.75rem;
+  transform: inherit;
+  height: auto;
+  width: auto;
+  padding: var(--space-2px);
+}
+
+.block.app-download .store a {
+  display: block;
```
[the remark this answers is not in the document]
Comment (ravverma) [reply blocks/app-download/app-download.css:51]: it is needed for the mobile view.

File: blocks/app-download/app-download.css (hunk 1 above)
Comment (ravverma) [reply blocks/app-download/app-download.css:28]: done

**Comment by JiangLong2019** [comment]
Seems the mobile view is different from the original site.
Original site:
image

EDS:
image

**Review by anagarwa — APPROVED** [review]

**Comment by ravverma** [comment]
> Seems the mobile view is different from the original site. Original site: image
> 
> EDS: image

did it on purpose as the old one looks distorted.

**Comment by vivesing** [comment]
image
QR code shadow is missing which is there in the original site, in mobile view

**Comment by vivesing** [comment]
In smaller width, padding is missing along with alignment of buttons 
PR: 
image
Original:
image

File: styles/styles.css [hunk 3]
```diff
@@ -487,6 +488,11 @@ main .section.shadow-gray {
   box-shadow: inset 0 1.25rem 2.5rem var(--background-color-grey);
 }
 
+main .section.app-download {
```
[the remark this answers is not in the document]
Comment (ravverma) [reply styles/styles.css:491]: I should rename it to more generic I think as the the style is full width and in section we have not full width.

**Comment by ravverma** [comment]
> In smaller width, padding is missing along with alignment of buttons PR: image Original: image

fixed.

**Comment by ravverma** [comment]
> image QR code shadow is missing which is there in the original site, in mobile and desktop view

fixed

File: styles/styles.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (ravverma) [reply styles/styles.css]: done

**Pull request merged**